docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -89,54 +90,62 @@ OpenSearch Python client offers a helper called bulk() which allows us to send m
 
     python epicurious_recipes_import.py
 
-Sample queries with HTTP client
--------------------------------
 
-With the data in place, we can start trying some queries against your OpenSearch service. Since it has a simple HTTP interface, you can use your favorite HTTP client. In these examples, we will use `httpie <https://github.com/httpie/httpie>`_ because it's one of our favorites.
+.. _get-mapping-with-python:
 
-First, export the ``SERVICE_URI`` variable with your OpenSearch service URI address and index name from the previous script:
+Get data mapping with Python
+----------------------------
+When we sent the data to OpenSearch cluster on :ref:`load the data with Python <load-data-with-python>`, no data structure was specified. 
+As default, OpenSearch uses **dynamic mapping** to automatically add the fields to their type mapping. 
 
-.. code:: bash
+Use ``get_mapping`` method with the index name as the argument to check the mapping definition of your data.
 
-    export SERVICE_URI="YOUR_SERVICE_URI_HERE/epicurious-recipes"
+.. code:: python
 
-1. Execute a basic search for the word ``vegan`` across all documents and fields:
+    import pprint
 
-.. code:: bash
+    INDEX_NAME = 'epicurious-recipes'
+    mapping_data = os_client.indices.get_mapping(INDEX_NAME)
 
-    http "$SERVICE_URI/_search?q=vegan"
+    # Find index doc_type
+    doc_type = list(mapping_data[INDEX_NAME]["mappings"].keys())[0]
 
-2. Search for ``vegan`` in the ``desc`` or ``title`` fields only: 
+    schema = mapping_data[INDEX_NAME]["mappings"][doc_type]
+    print(f"Fields: {list(schema.keys())} \n")
+    print(schema)
 
-.. code:: bash
+You should be able to see this output:
 
-    http POST "$SERVICE_URI/_search" <<< '
-    {
-        "query": {
-            "multi_match": {
-                "query": "vegan",
-                "fields": ["desc", "title"]
-            }
-        }
-    }
-    '
+.. code-block:: bash
 
-3. Search for recipes published only in 2013:
+    Fields: ['calories', 'categories', 'date', 'desc', 'directions', 'fat', 'ingredients', 'protein', 'rating', 'sodium', 'title'] 
 
-.. code:: bash
+And the mapping with the fields and their respective types.
+
+.. code-block:: bash
+
+    {'calories': {'type': 'float'},
+    'categories': {'fields': {'keyword': {'ignore_above': 256, 'type': 'keyword'}},
```

Comment:
The formatting still looks chaotic. Maybe arrange it a bit manually? so at least the main properties are aligned and it is easier to read it
<img width="763" alt="Screenshot 2022-02-23 at 17 11 19" src="https://user-images.githubusercontent.com/4600541/155359691-124e1bca-e899-478d-b44a-303ddeb238c8.png">

Reply:
I adjusted manually
<img width="817" alt="Screenshot 2022-02-23 at 17 20 02" src="https://user-images.githubusercontent.com/36624597/155363223-0c1b5dae-2a7e-4be3-83a8-22fe26805f1c.png">

do you think is better?

Reply:
this is better! The normal format is still way more readable - like here - https://en.wikipedia.org/wiki/JSON. I suspect we want to save some space, since this JSON object is long when formatted properly. There are also online formatters that can help.

Reply:
I see it. I think it was something from the command line looked fine but copying and pasting, it somehow broke the formatting. Please, would be possible to check how it is now? I want to avoid the long json schema because it would take too much space of the page.